src/patternfly/components/Content/content.scss

```diff
@@ -239,9 +257,12 @@
 
   ol {
     padding-left: var(--pf-c-content--ol--PaddingLeft);
-    margin-top: var(--pf-c-content--ol--MarginTop);
     margin-left: var(--pf-c-content--ol--MarginLeft);
 
+    &:not(:first-child) {
+      margin-top: var(--pf-c-content--ol--MarginTop);
```

Comment:
Can you think of a reason why we would want this?  One thing this does is cause a heading + `<ol>` (or `<ul>`) have a `16px` spacer where a heading + `<p>` will have an `8px` spacer. Do we want that?

It would also allow users to use a different spacer above lists by styling the list, versus having to style whatever it is that precedes the list. But we don't allow that with the other elements - `<p>`'s, for example.

Reply:
Agreed I'm not sure if we even need the margin-top because any of the elements in `pf-c-content` will have a margin-bottom if they're not the last element.

@mcarrano @kybaker @mceledonia do any of you remember why there was a margin-top of 16px set on the `<ol>` and `<ul>` when there is already a margin-bottom of 8px set on any element on top of it. Do you know if there was a reason behind having larger spacing on top of the lists?

Reply:
@mcoker @mattnolting after speaking offline with Matt and Michael, we decided that `<ol>` and `<ul>` would only be placed after a `heading` or a `<p>` element therefore the margin-bottom on the later elements would be sufficient spacing. (So we can remove the margin-top  on `<ol>` and `<ul>`). They also said that they would prefer to see 8px space between header and `<ol>` and `<ul>` elements so we don't need to introduce a new variable to replace the margin-top vars which had a size of 16px.